code/datums/ai/basic_mobs/targetting_datums/basic_targetting_datum.dm

```diff
@@ -35,6 +37,9 @@
 		var/mob/living/L = the_target
 		if(faction_check(living_mob, L) || L.stat)
 			return FALSE
+		else if(check_hostile)
+			if(living_mob.maxHealth == living_mob.health)
+				return FALSE
```

Comment:
This was awful behavior of it becoming hostile to everything on old retaliate mobs, fuck that